EZSwiftExtensionsTests/UIViewTests.swift

```diff
@@ -60,5 +60,58 @@ class UIViewTests: XCTestCase {
         view.removeSubviews()
         XCTAssertEqual(view.subviews.count, 0)
     }
+    
+    func testRoundView() {
+        let wideView = UIView(x: 0, y: 0, w: 200, h: 80)
+        XCTAssertEqual(wideView.layer.cornerRadius, 0)
+        XCTAssertEqual(wideView.layer.masksToBounds, false)
+        
+        wideView.roundView()
+        XCTAssertEqual(wideView.layer.cornerRadius, CGFloat(40.0))
+        XCTAssertEqual(wideView.layer.masksToBounds, true)
+        
+        let tallView = UIView(x: 0, y: 0, w: 30, h: 300)
+        XCTAssertEqual(tallView.layer.cornerRadius, 0)
+        XCTAssertEqual(tallView.layer.masksToBounds, false)
+        
+        tallView.roundView()
+        XCTAssertEqual(tallView.layer.cornerRadius, CGFloat(15.0))
+        XCTAssertEqual(tallView.layer.masksToBounds, true)
+    }
+    
+    func testRoundViewWithBorder() {
+        let view = UIView(x: 0, y: 0, w: 100, h: 100)
+        let color = UIColor.blue
+        let borderWidth: CGFloat = 5.0
+        view.layer.borderColor = nil // override default view.layer.borderColor
```

Comment:
I am not wholly comfortable with this nil set and then assert. Maybe consider optional bordercolor as arg or something. Then asset the content of the optional to find nothing there.